app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md

```diff
@@ -8,44 +8,82 @@ related_content:
 
-- a different but related degree to the subject you want to teach
-- an A level but not a degree in the subject
-- an unrelated degree but relevant professional experience
-- studied for a languages degree, but need to develop a second language to an acceptable level for teaching in schools 
-- not used your subject knowledge for a while
+* finished your degree more than 5 years ago
+* a different but related degree to the subject you want to teach
+* an A level but not a degree in the subject you want to teach
+* an unrelated degree but relevant professional experience
+* studied for a languages degree, but need to develop a second language to an acceptable level for teaching in schools
 
-When you apply for teacher training, your teacher training provider will help you establish whether you need to improve your subject knowledge. They may decide to make completing a SKE course a condition of your offer to train with them.
+## How to do an SKE course
 
-SKE courses are currently offered in 9 secondary subjects:  
+When you apply for teacher training in England, talk to your training provider if you think you need to improve your subject knowledge. They’ll help you decide if you need to complete an SKE course, and may make it a condition of their offer to train with them.
 
-- biology 
-- chemistry 
-- computing 
-- design and technology 
-- English
-- languages 
-- mathematics
-- physics 
-- religious education
+You’ll choose your SKE course provider, but your teacher training provider can help you decide on the right one for you.
 
-You can also do a SKE course in primary mathematics.
+### SKE subjects
 
-SKE course lengths can range from 8 to 28 weeks. They're usually online, but some can be classroom based and some can be a mixture of both. You can choose the provider you want to complete your SKE course with.
+SKE courses are currently available in 9 secondary subjects:
 
-Courses are available across England. Most candidates complete a SKE course before starting their teacher training year, but it’s sometimes possible to complete it alongside your initial teacher training (ITT) course. You may be able to complete the SKE course on a part-time basis.
+* biology
+* chemistry
+* computing
+* design and technology
+* English
+* languages
+* mathematics
+* physics
+* religious education
 
-You may also be eligible for a bursary of £175 per week if you train to teach in the 2022/23 academic year. This will go towards any extra costs you may have during the SKE course. Contact your ITT and SKE providers for more information.  
+You can also do one in primary mathematics.
 
-[Find out if there's a subject knowledge enhancement (SKE) course for your subject](https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory).
+$ske-course-video$
+
+## How does an SKE course work?
+
+They're usually online, but some can be classroom based and some can be a mixture of both. 
```

Comment:
```suggestion
SKE courses are usually online, but some can be classroom based and some can be a mixture of both. 
```

Reply:
I just changed this here because I felt like I was saying 'SKE courses' too many times (also not very SEO friendly to have the same phrase repeated so much)